lib/standalone.js

```diff
@@ -146,7 +145,7 @@ module.exports = function (options) {
 						return stylelint._createStylelintResult(postcssResult);
 					});
 			})
-			.catch(_.partial(handleError, stylelint))
+			.catch(handleError.bind(null, stylelint))
```

Comment:
[suggestion] IMO, it seems straightforward to me to call just `handleError()` here, instead of using `.bind()`. What do you think?

```suggestion
			.catch((error) => {
				return handleError(stylelint, error);
			})
```

Reply:
Hi @ybiquitous, that's a good idea. I did this in 94135f04.

Reply:
Thank you! 😄